frontend/src/services/requestService.js

```diff
@@ -0,0 +1,62 @@
+import axiosService from 'axios';
+
+const axios = axiosService.create {
+    headers: {
+        'X-Requested-With': 'XMLHttpRequest',
+        'Content-type': 'application/json',
+        'Accept': 'application/json'
+    }
+};
+
+const token = 'getTokenFromAuthService';
```

Comment:
token should be used dynamically. Use interceptors

https://github.com/axios/axios#interceptors